config/config.go

```diff
@@ -228,6 +228,12 @@ type Local struct {
 	// Generate ProcessBlockMetrics telemetry event
 	EnableProcessBlockStats bool `version[0]:""`
 
+	// Generate AccountUpdates telemetry event
+	EnableAccountUpdatesStats bool `version[17]:"false"`
+
+	// Frequency for generating accountUpdates telemetry event
+	AccountUpdatesStatsFrequency time.Duration `version[17]:"5"`
```

Comment:
The value here is not in the correct units. The `time.Duration` is given in units of nanoseconds, so you'd need to specify a value of `5000000000` for 5 seconds. ( which are a good interval )
Also, given that we're using this as an interval, maybe we should change the name of this field to `AccountUpdatesStatsInterval`